CONTRIBUTING.md

```diff
@@ -0,0 +1,13 @@
+# Prerequisites
+
+-   have `npm` installed and in path
+-   have `git` installed and in path
+
+# Setting up the repo for contributing
```

Comment:
I don't think the `\*nix` is relevant.

Reply:
i dont use windows (or haiku or whatever) so i have no knowledge of non *nix
"... for *nix" does not mean not for windows.  it simply means that this works for *nix. and does not specify anything for windows, etc